layouts/_default/baseof.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="{{.Site.Language.Lang}}">
```

Comment:
```suggestion
<html lang="{{ .Site.Language.Lang }}">
```

Thanks for the PR, I'm going to be very picky and insist the style here matches the rest of the codebase 🙂

Reply:
fair point, thank you! fixed!